README.md

```diff
@@ -159,20 +159,43 @@ When having CSV/TSV input, you can also decide to provide multiple sequence colu
 
 ### Examples of running:
 
+If you want to run the tool on example datasets, please clone the repository first:
```

Comment:
I would add an explanation why they should clone the repository - to access example data.